homepage/templates/homepage/how_cute_404.html

```diff
@@ -46,20 +46,20 @@
 </style>
 <html>
 	<head>
-	   <title>404 :: Guya.moe</title>
-	<meta name="twitter:description" content="A place to read the Kaguya-sama: Love is War (Kaguya Wants to Be Confessed To) manga. No ads. No bad reader. All guya." />
-	<meta property="og:description" content="A place to read the Kaguya-sama: Love is War (Kaguya Wants to Be Confessed To) manga. No ads. No bad reader. All guya.">
-	<meta name="description" content="A place to read the Kaguya-sama: Love is War (aWants to Be Confessed To) manga. No ads. No bad reader. All guya.">
+		<title>{{ brand.name }} :: 404</title>
+		<meta name="twitter:description" content="{{ brand.description }}" />
+		<meta property="og:description" content="{{ brand.description }}">
+		<meta name="description" content="{{ brand.description }}">
 
 	</head>
 	<body>
-    	<img src="https://guya.moe/static/img/how_cute_404.png" />
-    	<content>
-    		<p>You were trying to visit a page<br> that doesn't even exist?
-    		<p>How cute.
-    		<p>404
-    		<p>NOT FOUND
-    		<a href="/">Return to main page</a>
-    	</content>
+			<img src="{% static 'how_cute_404.png' %}" />
```

Comment:
This is untested.

Reply:
I believe this will need to be `{% static 'img/how_cute_404.png' %}`